psydac/api/tests/test_2d_multipatch_mapping_poisson.py

```diff
@@ -281,21 +277,9 @@ def test_poisson_2d_4_patch_dirichlet_0():
     D3     = mapping_3(C)
     D4     = mapping_4(D)
 
-    domain1 = D1.join(D2, name = 'domain1',
-                bnd_minus = D1.get_boundary(axis=1, ext=1),
-                bnd_plus  = D2.get_boundary(axis=1, ext=-1))
-
-    domain2 = D3.join(D4, name='domain2',
-                    bnd_minus = D3.get_boundary(axis=1, ext=1),
-                    bnd_plus  = D4.get_boundary(axis=1, ext=-1))
-
-    domain = domain1.join(domain2, name='domain',
-                    bnd_minus = D1.get_boundary(axis=0, ext=1),
-                    bnd_plus  = D3.get_boundary(axis=0, ext=-1))
-
-    domain = domain.join(domain, name='domain',
-                        bnd_minus = D2.get_boundary(axis=0, ext=1),
-                        bnd_plus  = D4.get_boundary(axis=0, ext=-1))
+    connectivity = [((0,1,1),(1,1,-1)), ((2,1,1),(3,1,-1)), ((0,0,1),(2,0,-1)),((1,0,1),(3,0,-1))]
+    domains = [D1, D2, D3, D4]
+    domain = Domain.join(domains, connectivity, 'domain')
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)